man/swaync.5.scd

```diff
@@ -595,7 +604,8 @@ config file to be able to detect config errors
 		},
 		"mpris": {
 			"image-size": 96,
-			"image-radius": 12
+			"image-radius": 12,
+			"blacklist": ["playerctld"]
```

Comment:
Wouldn't this be `org.mpris.MediaPlayer2.playerctld`? That's at least what appears on my machine

Reply:
Or is this due to the blacklist checking with regex?

Reply:
yeah, because regex